README.md

```diff
@@ -110,6 +110,23 @@ docker restart valhalla
 
 If you change the PBF files by either adding new ones or deleting any, Valhalla will build new tiles on the next restart unless told not to (e.g. setting `use_tiles_ignore_pbf=True`).
 
+#### Build Valhalla Graph with custom elevation tiles
+
+Elevation tiles need to be in HGT (file format of the SRTM dataset) format and need to be named like `NXXEYYY.hgt`. [More info about format](https://github.com/tilezen/joerd/blob/master/docs/formats.md#skadi).
+
+You need to store elevation tiles in `custom_files/elevation_data/` directory. Tiles need to be grouped in folders by latitude , for example:
+
+```
+custom_files/elevation_data/
+  N53/
+    N53E016.hgt
+    N53E017.hgt
+  N54/
+    N54E016.hgt
+```
+
+After providing new elevation tiles, you need to restart container and rebuild graph with `force_rebuild=True` environment variable. 
```

Comment:
doesn't it work to create the `custom_files/elevation_data` directory, move the tiles there and only then start a container? (that is instead of rebuilding an existing graph, which is quite a waste of computation)

Reply:
Elevation API just needs to restart a container and it will use new tiles. What about Routing API? I assumed we need to rebuild to use new elevation tiles.

Reply:
you only need to rebuild if you already had a graph before you got new elevation data. I'd assume if someone has alternative elevation data they'd like to use that right away. you do say that, I'd just make that more explicit, how about 

"if you had an existing graph before you acquired new elevation data, you'll need to rebuild the graph for the new data to become available, e.g. by starting a new container with `force_rebuild=True`"

Reply:
Yup, that makes sense.